cmd/main.go

```diff
@@ -112,7 +113,9 @@ func main() {
 	)
 
 	foundProject, version := getProjectVersion()
-	if foundProject && version == "1" {
+	if foundProject && version == project.Version1 {
```

Comment:
Cobra command library does provide mechanism to mark commands [as deprecated](https://godoc.org/github.com/spf13/cobra#Command) but I think the implemented approach will work better for our case.